### PR TITLE
Added help message for KIA0106 Service Account not found for this principal.

### PR DESCRIPTION
**Pull request by hhovsepy**
RFE https://github.com/kiali/kiali/issues/4424

Feature done in https://github.com/kiali/kiali/pull/5057

### CODE REVIEW

File: content/en/docs/Features/validations.md [hunk 1]
```diff
@@ -91,6 +91,34 @@ Either remove the host from the list, correct if there is any typo or deploy a n
 - [Service association requirement](https://istio.io/docs/ops/deployment/requirements)
 
 
+### KIA0106 - Service Account not found for this principal
+
+AuthorizationPolicy has a Source field, where specifies the source identities of a request.
+In a Source field it accepts the principals, a list of peer identities derived from the peer certificate. The peer identity is in the format of "<TRUST_DOMAIN>/ns/<NAMESPACE>/sa/<SERVICE_ACCOUNT>", for example, "cluster.local/ns/default/sa/productpage".
```
Comment (jshaughn) [inline content/en/docs/Features/validations.md]:
```suggestion
In a Source field it accepts the principals, a list of peer identities derived from the peer certificate. The peer identity is in the format of `<TRUST_DOMAIN>/ns/<NAMESPACE>/sa/<SERVICE_ACCOUNT>`, for example, `cluster.local/ns/default/sa/productpage`.
```

Comment (jshaughn) [reply content/en/docs/Features/validations.md]: Hayk, the current PR renders a double-slash, I think you want to make a small change, here.

**Review by jshaughn — CHANGES_REQUESTED** [review]

**Review by jshaughn — APPROVED** [review]
LG!

**Review by jshaughn — COMMENTED** [review]
Note that this PR is targeting 'current'.  Did you mean to target staging?

**Comment by hhovsepy** [comment]
oops, thank you @jshaughn, have retargeted

**Pull request merged**